boards/arc/arduino_101_sss/arduino_101_sss.yaml

```diff
@@ -0,0 +1,12 @@
+identifier: arduino_101_sss
+name: arduino_101_sss
```

Comment:
we have cases where might have multiple board definitions inside a board directory, so that already does not work.